categorize_adjectives/bert_similarity_calc.py

```diff
@@ -0,0 +1,60 @@
+"""Generate dictionary of distances between each pair of adjective"""
+import os
+import pickle
+from libraries.bert_embeddings import BertEmbedding
+from scipy.spatial.distance import cosine
+
+def get_most_frequent_adjs(noun_to_adj, num_adjs):
+    """Generate the list of most frequent num_adjs in noun_to_adj dictionary"""
+    selected_adj = []
+    adj_count = {}
+    for noun in noun_to_adj.keys():
+        for adj, _ in noun_to_adj[noun]:
+            try:
+                adj_count[adj] += -1
```

Comment:
This is a little confusing adding -1 for counting. Can you add a comment or change it to +1 and sort it in desc order?